cmd/cluster.go

```diff
@@ -18,14 +18,20 @@ func setupClusterCommand() *cobra.Command {
 				return cobraext.FlagParsingError(err, cobraext.DaemonModeFlagName)
 			}
 
-			err = cluster.BootUp(daemonMode)
+			stackVersion, err := cmd.Flags().GetString(cobraext.StackVersionFlagName)
+			if err != nil {
+				return cobraext.FlagParsingError(err, cobraext.StackVersionFlagName)
+			}
+
+			err = cluster.BootUp(daemonMode, stackVersion)
 			if err != nil {
 				return errors.Wrap(err, "booting up the cluster failed")
 			}
 			return nil
 		},
 	}
 	upCommand.Flags().BoolP(cobraext.DaemonModeFlagName, "d", false, cobraext.DaemonModeFlagDescription)
+	upCommand.Flags().StringP(cobraext.StackVersionFlagName, "", cluster.DefaultVersion, cobraext.StackVersionDescription)
 
 	downCommand := &cobra.Command{
```

Comment:
I did not test this but I wonder if down works as expected if non default values for the stack version are used.